### PR TITLE
Explain how to access to File share dirs.

### PR DESCRIPTION
**Pull request by manuelesimi**
A clarification about how to access to File share dirs in the Azure doc.

### CODE REVIEW

File: docs/azure.rst [hunk 1]
```diff
@@ -80,8 +80,8 @@ For example::
 		}
 	}
 
-The files available in the File share will be available to the task in the directory:
-<YOUR MOUNT DESTINATION>/<YOUR SOURCE FILE SHARE NAME>.
+The files in the File share are available to the task in the directory:
+`<YOUR MOUNT DESTINATION>/<YOUR SOURCE FILE SHARE NAME>`.
```
Comment (pditommaso) [inline docs/azure.rst:85]: Maybe it should be added a basic example to make it more intuitive.

Comment (manuelesimi) [reply docs/azure.rst:85]: See the lines below :)

Comment (manuelesimi) [reply docs/azure.rst:85]: https://github.com/nextflow-io/nextflow/blob/acb17bbff46ac60f8aebf160b2336a9437ef23f8/docs/azure.rst#L86-L100

**Review by pditommaso — APPROVED** [review]

**Comment by pditommaso** [comment]
Great. thanks

**Pull request merged**